src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -270,8 +270,19 @@ <h3>Headers</h3>
     <!-- ko if: hasBody -->
     <h3>Body</h3>
 
-    <div class="flex justify-content-end">
-        <!-- ko if: request.readonlyBodyFormat -->
+    <div class="row flex flex-row" style="margin-bottom: 3%;">
+        <div class="col-4">
+            <!-- ko if: request.bodyFormat() === 'raw' -->
+            <select id="requestRepresentations" class="form-control" style="width: auto;" data-bind="value: $component.selectedRepresentation">
+                <!-- ko foreach: { data: request.representations, as: 'representation' } -->
+                <option data-bind="value: representation, text: 'Example ' + ($index() + 1)"></option>
```

Comment:
`'Example ' + ($index() + 1)` - code samples come with a name which is reflected in operation details and hence the customers would expect to see same names in this dropdown.

Reply:
it is the field called "typeName" which is not a required field so the user can still leave it empty. I updated the code as - using the name if exists, if not I am adding Example with its index. So that users can easily spot named representations and also not named ones.

![Uploading Screen Shot 2021-11-10 at 12.53.28.png…]()

Reply:
As far as I understand, there can be just one empty, which you may name just "Example" or "Default".

Reply:
Also, if you use rely on ` $index()` here, you may have situation like this: ["MyExample", "AnotherExample", "Example 3"] - the numbering will be off.

Reply:
No, It can be multiple unnamed representations. So that is why that made sense to me with index. ["Example 1", "my named example", "Example 3"]. What do you suggest here?

Reply:
Yep, saw the case you sent me offline, makes sense then.